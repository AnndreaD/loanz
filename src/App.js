import React from 'react';
import LoanPage from './pages/LoanPage';
import './App.scss';

function App() {
    return (
        <div className="app">
            <LoanPage />
        </div>
    );
}

export default App;
