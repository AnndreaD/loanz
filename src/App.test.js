import React from 'react';
import renderer from 'react-test-renderer';
import App from './App';

describe('App', () => {
    it('SNAPSHOT: should render expected content', () => {
        const defaultTree = renderer.create(<App />);
        expect(defaultTree.toJSON()).toMatchSnapshot();
    });
});
