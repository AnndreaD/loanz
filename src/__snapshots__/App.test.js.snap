// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App SNAPSHOT: should render expected content 1`] = `
<div
  className="app"
>
  <div
    className="container"
  >
    <h1
      className="container-title"
    >
      Lånekalkulator
    </h1>
    <div
      className="slider-container"
    >
      <p
        className="MuiTypography-root MuiTypography-body1 MuiTypography-gutterBottom"
        id="discrete-slider-small-steps"
      >
        Hvor mye ønsker du å låne?
      </p>
      <div
        className="MuiFormControl-root MuiTextField-root"
      >
        <div
          className="MuiInputBase-root MuiOutlinedInput-root MuiInputBase-formControl"
          onClick={[Function]}
        >
          <input
            aria-invalid={false}
            autoFocus={false}
            className="MuiInputBase-input MuiOutlinedInput-input"
            disabled={false}
            id="outlined-basic"
            onAnimationStart={[Function]}
            onBlur={[Function]}
            onChange={[Function]}
            onFocus={[Function]}
            required={false}
            type="text"
            value={1000}
          />
          <fieldset
            aria-hidden={true}
            className="PrivateNotchedOutline-root-1 MuiOutlinedInput-notchedOutline"
            style={
              Object {
                "paddingLeft": 8,
              }
            }
          >
            <legend
              className="PrivateNotchedOutline-legend-2"
              style={
                Object {
                  "width": 0.01,
                }
              }
            >
              <span
                dangerouslySetInnerHTML={
                  Object {
                    "__html": "&#8203;",
                  }
                }
              />
            </legend>
          </fieldset>
        </div>
      </div>
      <div
        className="loan-slider"
      >
        <span
          className="MuiSlider-root MuiSlider-colorPrimary MuiSlider-marked"
          onMouseDown={[Function]}
        >
          <span
            className="MuiSlider-rail"
          />
          <span
            className="MuiSlider-track"
            style={
              Object {
                "left": "0%",
                "width": "0%",
              }
            }
          />
          <input
            type="hidden"
            value="1000"
          />
          <span
            className="MuiSlider-mark MuiSlider-markActive"
            data-index={0}
            style={
              Object {
                "left": "-0.010001000100010001%",
              }
            }
          />
          <span
            aria-hidden={true}
            className="MuiSlider-markLabel MuiSlider-markLabelActive"
            data-index={0}
            style={
              Object {
                "left": "-0.010001000100010001%",
              }
            }
          >
            0
          </span>
          <span
            className="MuiSlider-mark"
            data-index={1}
            style={
              Object {
                "left": "100%",
              }
            }
          />
          <span
            aria-hidden={true}
            className="MuiSlider-markLabel"
            data-index={1}
            style={
              Object {
                "left": "100%",
              }
            }
          >
            10 000 000
          </span>
          <span
            aria-labelledby="discrete-slider-small-steps"
            aria-orientation="horizontal"
            aria-valuemax={10000000}
            aria-valuemin={1000}
            aria-valuenow={1000}
            className="MuiSlider-thumb MuiSlider-thumbColorPrimary PrivateValueLabel-thumb-5"
            data-index={0}
            onBlur={[Function]}
            onFocus={[Function]}
            onKeyDown={[Function]}
            onMouseLeave={[Function]}
            onMouseOver={[Function]}
            role="slider"
            style={
              Object {
                "left": "0%",
              }
            }
            tabIndex={0}
          >
            <span
              className="PrivateValueLabel-offset-7 MuiSlider-valueLabel"
            >
              <span
                className="PrivateValueLabel-circle-8"
              >
                <span
                  className="PrivateValueLabel-label-9"
                >
                  1000
                </span>
              </span>
            </span>
          </span>
        </span>
      </div>
    </div>
    <div
      className="slider-container"
    >
      <p
        className="MuiTypography-root MuiTypography-body1 MuiTypography-gutterBottom"
        id="discrete-slider-small-steps"
      >
        Over hvor mange år?
      </p>
      <div
        className="MuiFormControl-root MuiTextField-root"
      >
        <div
          className="MuiInputBase-root MuiOutlinedInput-root MuiInputBase-formControl"
          onClick={[Function]}
        >
          <input
            aria-invalid={false}
            autoFocus={false}
            className="MuiInputBase-input MuiOutlinedInput-input"
            disabled={false}
            id="outlined-basic"
            onAnimationStart={[Function]}
            onBlur={[Function]}
            onChange={[Function]}
            onFocus={[Function]}
            required={false}
            type="text"
            value={1}
          />
          <fieldset
            aria-hidden={true}
            className="PrivateNotchedOutline-root-1 MuiOutlinedInput-notchedOutline"
            style={
              Object {
                "paddingLeft": 8,
              }
            }
          >
            <legend
              className="PrivateNotchedOutline-legend-2"
              style={
                Object {
                  "width": 0.01,
                }
              }
            >
              <span
                dangerouslySetInnerHTML={
                  Object {
                    "__html": "&#8203;",
                  }
                }
              />
            </legend>
          </fieldset>
        </div>
      </div>
      <div
        className="loan-slider"
      >
        <span
          className="MuiSlider-root MuiSlider-colorPrimary MuiSlider-marked"
          onMouseDown={[Function]}
        >
          <span
            className="MuiSlider-rail"
          />
          <span
            className="MuiSlider-track"
            style={
              Object {
                "left": "0%",
                "width": "0%",
              }
            }
          />
          <input
            type="hidden"
            value="1"
          />
          <span
            className="MuiSlider-mark MuiSlider-markActive"
            data-index={0}
            style={
              Object {
                "left": "0%",
              }
            }
          />
          <span
            aria-hidden={true}
            className="MuiSlider-markLabel MuiSlider-markLabelActive"
            data-index={0}
            style={
              Object {
                "left": "0%",
              }
            }
          >
            1
          </span>
          <span
            className="MuiSlider-mark"
            data-index={1}
            style={
              Object {
                "left": "100%",
              }
            }
          />
          <span
            aria-hidden={true}
            className="MuiSlider-markLabel"
            data-index={1}
            style={
              Object {
                "left": "100%",
              }
            }
          >
            20
          </span>
          <span
            aria-labelledby="discrete-slider-small-steps"
            aria-orientation="horizontal"
            aria-valuemax={20}
            aria-valuemin={1}
            aria-valuenow={1}
            className="MuiSlider-thumb MuiSlider-thumbColorPrimary PrivateValueLabel-thumb-5"
            data-index={0}
            onBlur={[Function]}
            onFocus={[Function]}
            onKeyDown={[Function]}
            onMouseLeave={[Function]}
            onMouseOver={[Function]}
            role="slider"
            style={
              Object {
                "left": "0%",
              }
            }
            tabIndex={0}
          >
            <span
              className="PrivateValueLabel-offset-7 MuiSlider-valueLabel"
            >
              <span
                className="PrivateValueLabel-circle-8"
              >
                <span
                  className="PrivateValueLabel-label-9"
                >
                  1
                </span>
              </span>
            </span>
          </span>
        </span>
      </div>
    </div>
    <h3>
      Estimert totalpris 
      1032
       
      Kr
    </h3>
    <p>
      Rente 3,5% 
    </p>
    <p>
      Estimert månedlig avdrag 
       
      86
       
      Kr
    </p>
  </div>
</div>
`;
